docs/proposals/004-endpoint-picker-protocol/README.md

```diff
@@ -58,7 +60,16 @@ dynamicMetadata: {
 }
 ```
 
-### Why envoy.lb namespace as a default? 
+The endpoints specified in `x-gateway-destination-endpoint-fallback` MAY be tried after the endpoints specified in `x-gateway-destination-endpoint` if all the endpoints specified in `x-gateway-destination-endpoint` are unavailable.
```

Comment:
When `x-gateway-destination-endpoint` defines more than one `IP:Port`, the additional entries act as a fallback, correct? If so, why is `x-gateway-destination-endpoint-fallback` still needed?

Reply:
it's unnecessary, but not sure if it's ok or not to remove it directly from the proposal. I will removed it first.